apps/client/app/modules/explore/index.tsx

```diff
@@ -1,19 +1,38 @@
 import {Header} from '@/components/layout/index.ts'
 import {useInView} from 'react-intersection-observer'
 import {ExploreHeroSection} from './components/hero/index.tsx'
-import {WebTabComponent} from './components/tabs/web.tsx'
+import {categories, WebTabComponent} from './components/tabs/web.tsx'
 import {FadeIn, FadeInStagger} from '@/components/animation/FadeIn.tsx'
-import {Fragment} from 'react'
+import {Fragment, useEffect, useState} from 'react'
 import {Content} from '@/components/Content/index.tsx'
 import {MobileTabComponent} from './components/tabs/mobile.tsx'
 import {imageUrls} from '../landing-page/index.tsx'
 import {Footer} from '@/components/footer/index.tsx'
+import {EmptyState} from './components/empty-state/index.tsx'
 
 export const ExploreModule = () => {
   const {ref: heroRef, inView} = useInView({
     threshold: 0,
   })
 
+  const [empty, setEmpty] = useState(false)
+
+  useEffect(() => {
+    const dettyDecemberCategory = categories.find(
+      category => category.name === 'Detty December',
+    )
+
+    if (dettyDecemberCategory && dettyDecemberCategory.count.length <= 0) {
+      setEmpty(true)
+    } else {
+      setEmpty(false)
+    }
+  }, [categories])
```

Comment:
For now, we could just have a boolean somewhere to toggle the rendering of the empty state component. since all this will be determined by an API.

```suggestion
   const isEmpty = false;
```